setup.py

```diff
@@ -73,7 +72,7 @@ def setup_package():
     setup(
         name=package,
         version=versioneer.get_version(),
-        cmdclass=versioneer.get_cmdclass({"build_py": build_py}),
```

Comment:
Don't remember why we had this, any clue?

Reply:
May have been legacy PySAL stuff that we kept around when transitioning to `versioneer`: https://github.com/pysal/libpysal/pull/429

Reply:
It was giving me trouble over in `spopt` so I tried [getting rid of it](https://github.com/pysal/spopt/pull/340/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L37). Passed after that.

Reply:
okay :D